test/impl/builders/check_run_builder.py

```diff
@@ -15,9 +15,11 @@ def __init__(
         }
 
     def completed_at(
-        self, completed_at: Union[str, datetime]
+        self, completed_at: Union[str, datetime, None]
     ) -> Union["CheckRunBuilder", CheckRun]:
-        self.raw_check_run["completedAt"] = transform_datetime(completed_at)
+        self.raw_check_run["completedAt"] = (
+            transform_datetime(completed_at) if completed_at else None
```

Comment:
You shouldn't need to do this ternary because `transform_datetime` handles the `None` case (although perhaps accidentally). 

Just an fyi. Feel free to leave this here or update the arg type of transform datetime

Reply:
hm `transform_datetime` is used for other timestamps that we expect to be non-null so keeping this as is